snuba/query/data_source/simple.py

```diff
@@ -31,6 +31,7 @@ class Entity(SimpleDataSource):
 
     key: EntityKey
     schema: ColumnSet
+    sample: Optional[float] = None
```

Comment:
Will final be added here as well? Do you have any plan for `turbo`?

Reply:
Filippo is right about turbo, that is not part of the SnQL query syntax. It gets sent as a separate parameter in the request.

Reply:
Ok. Don't we need final here though since passing turbo (or indeed final itself) affects whether we run the data source with final

Reply:
Final is an attribute of the AST query right now. Maybe at some point it should also be part of the data source, but that is a change that can be made separately.